### PR TITLE
update readme en, test=document_fix

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

File: README_en.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+PaddleDetection implements varied mainstream object detection algorithms in modular design, and provides wealthy data augmentation methods, network components(such as backbones), loss functions, etc., and integrates abilities of model compression and cross-platform high-performance deployment.
+
+After a long time of industry practice polishing, PaddleDetection has had smooth and excellent user experience, it has been widely used by developers in more than ten industries such as industrial quality inspection, remote sensing image object detection, automatic inspection, new retail, Internet, and scientific research.
+
+<div align="center">
+  <img src="static/docs/images/football.gif" width='800'/>
+</div>
+
+### Product dynamic
```
Comment (qingqing01) [inline README_en.md]: Product News?

File: README_en.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+### Features
+
+- **Rich Models**
+PaddleDetection provides rich of models, including **100+ pre-trained models** such as **object detection**, **instance segmentation**, **face detection** etc. It covers a variety of **global competition champion** schemes.
+
+- **Use Concisely**
+Modular design, decouple each network component, developers easily build and try various detection models and optimization strategies, quickly get high-performance, customized algorithm.
+
+- **Getting Through End to End**
```
Comment (qingqing01) [inline README_en.md]: 28, 31行标题翻译不太准确

File: README_en.md (hunk 1 above)
Comment (jerrywgz) [reply README_en.md]: done, thx

File: README_en.md (hunk 2 above)
Comment (jerrywgz) [reply README_en.md]: done, thx

**Review by qingqing01 — APPROVED** [review]

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**